node/directives/checkUserAccess.ts

```diff
@@ -3,6 +3,63 @@ import type { GraphQLField } from 'graphql'
 import { defaultFieldResolver } from 'graphql'
 import { SchemaDirectiveVisitor } from 'graphql-tools'
 
+export async function checkUserOrAdminTokenAccess(
+  ctx: Context,
+  operation?: string
+) {
+  const {
+    vtex: { adminUserAuthToken, storeUserAuthToken, logger },
+    clients: { identity, vtexId },
+  } = ctx
+
+  if (!adminUserAuthToken && !storeUserAuthToken) {
+    logger.warn({
+      message: `CheckUserAccess: No admin or store token was provided for ${operation}`,
```

Comment:
Since you're already adding `operation` as a variable to be logged, you could remove it from the message text, right? I remember reading about logging best practices that having fixed log messages (without a dynamic part) is generally preferred, as that helps with log metrics, for example. Having the operation as a field should be enough for analyzing that value.

Reply:
Yes, I know, but just a reminder that this is an IO app, so it isn't easy to add a new field to index. If everybody wants to put a new field, there is a [problem](https://vtex.slack.com/archives/C02MGEZ392R/p1700681542688149?thread_ts=1700677521.753579&cid=C02MGEZ392R).